src/rpc/rawtransaction.cpp

```diff
@@ -162,6 +162,61 @@ static std::vector<RPCArg> CreateTxDoc()
     };
 }
 
+// Update PSBT with information from the mempool, the UTXO set, and the provided descriptors
+PartiallySignedTransaction ProcessPSBT(const JSONRPCRequest& request, const HidingSigningProvider provider)
```

Comment:
In 58f265f4de0051bbecb0f7e367a39258b24d5a40 "rpc: extract psbt updating logic into ProcessPSBT()"

I think it would be better to give the `UniValue` for the PSBT and a `NodeContext` rather than passing in the entire `JSONRPCRequest`.

Reply:
Done